packages/compass-aggregations/src/typings.d.ts

```diff
@@ -1 +1,4 @@
 declare module '@mongodb-js/mongodb-redux-common/app-registry';
+declare module 'mongodb-ace-autocompleter';
+declare module 'mongodb-query-parser';
+declare module 'semver';
```

Comment:
At least for semver is it possible to install [type definitions](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/semver) instead of completely ignoring the types of this package?